examples/src/main/java/org/apache/spark/examples/ml/JavaOneVsRestExample.java

```diff
@@ -75,7 +75,7 @@ public static void main(String[] args) {
 
     // compute the classification error on test data.
     double accuracy = evaluator.evaluate(predictions);
-    System.out.println("Test Error : " + (1 - accuracy));
+    System.out.println("Test Error = " + (1 - accuracy));
```

Comment:
Yeah, I was just trying to make things like this consistent with other similar examples.  I think I just saw "=" used more often, but it really doesn't make a difference to me.